lab-taylor/gulpfile.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+gulp.task('runTests', () => {
```

Comment:
This task doesn't properly run mocha, the test you have hangs and then fails due to a timeout error.